src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateGeneratorTestBase.cs

```diff
@@ -205,18 +205,115 @@ public static IEndpointRouteBuilder MapTestEndpoints(this IEndpointRouteBuilder
         return app;
     }
 
+    public enum TodoStatus
```

Comment:
Nit: Might want to move this type and others to `SharedTypes` and reference in test code using `typeof(TodoStatus)`. The codebase currently does this for the `TestService` type and https://github.com/dotnet/aspnetcore/pull/46715 updates the `Todo` type to follow this pattern.

Reply:
I agree we should do this, but I am going to put it in a separate PR since it causes enough churn in existing tests that I just want it to stand independently. Also means that Ihave to move from using the `[InlineData]` attribute to `[MemberData[` because I need to use type names in the Inline data.